users/static/base.css

```diff
@@ -34,7 +34,7 @@
   cursor: pointer;
 
   opacity: 0; /* hide this */
-  z-index: 2; /* and place it over the hamburger */
+  z-index: 8; /* and place it over the hamburger */
```

Comment:
Can you please remove this comment if it is no longer needed?

Reply:
done